src/Data/Foreign/NullOrUndefined.purs

```diff
@@ -28,3 +28,5 @@ unNullOrUndefined (NullOrUndefined m) = m
 readNullOrUndefined :: forall a. (Foreign -> F a) -> Foreign -> F (NullOrUndefined a)
 readNullOrUndefined _ value | isNull value || isUndefined value = pure (NullOrUndefined Nothing)
 readNullOrUndefined f value = NullOrUndefined <<< Just <$> f value
+
+foreign import writeNull :: Foreign
```

Comment:
ah, yeah. i ran into an issue with not being able to read `undefined` as JSON though, and the test i've pushed gives me `[null]` for `[NullOrUndefined (Nothing :: Maybe String)]`...